src/Utils/BreakingChangesFinder.php

```diff
@@ -269,9 +269,6 @@ private static function isChangeSafeForObjectOrInterfaceField(
         return false;
     }
 
-    /**
```

Comment:
this should imo at least stay

Reply:
Ah, right--forgot to restore this when I undid the array shapes stuff.